Prolangle/Languages/Bash.cs

```diff
@@ -0,0 +1,31 @@
+using Prolangle.Languages.Framework;
+
+namespace Prolangle.Languages;
+
+public class Bash : BaseLanguage
+{
+	private Bash()
+	{
+	}
+
+	public static Bash Instance { get; } = new();
+
+	public override Guid Id { get; } = Guid.NewGuid();
+	public override string Name { get; } = "Bash";
+
+	public override TypeSystem Typing { get; } =
+		TypeSystem.Weak | TypeSystem.Dynamic;
+
+	public override bool Compiled { get; } = false;
+	public override MemoryManagement MemoryManagement { get; } = MemoryManagement.TracingGarbageCollection;
+	public override SyntaxStyle SyntaxStyle { get; } = SyntaxStyle.Other;
```

Comment:
Are there other languages we might add that share a syntax style with bash? What would be a good name?

Reply:
Maybe `Shell`?

Reply:
I mean… `PowerShell` and `Nushell` are shells, but look quite different (as does DOS/`.bat`).

`UnixShell`?

Reply:
Is there another language with a similar syntax

Reply:
Many of bash's siblings are like it. `tcsh`, `zsh`, and so on. `fish` is somewhat different.

Reply:
I'm not sure we'll be adding any other language with that style soon. I changed my mind, maybe we should leave it as is.

Reply:
Right.